src/main/java/uk/gov/hmcts/darts/common/util/FileContentChecksum.java

```diff
@@ -29,10 +34,20 @@ public String calculate(byte[] bytes) {
         return encodeToString(md5(bytes));
     }
 
+    /**
+     * Calculates the digest when the source data has already been consumed.
+     */
     public String calculate(DigestInputStream digestInputStream) throws IOException {
         return encodeToString(digestInputStream.getMessageDigest().digest());
     }
 
+    @SneakyThrows
+    public String calculateFromFile(Path filePath) {
+        ByteSource byteSource = com.google.common.io.Files.asByteSource(filePath.toFile());
+        HashCode hc = byteSource.hash(Hashing.md5());
+        return encodeToString(hc.asBytes());
+    }
```

Comment:
You could follow the same code that generates the hashcode as is in the method FileContentChecksumTest.calculateFromInputStreamUsingAudioFileFromAzure for the file without the need to use another library

Reply:
I agree. We dont have to add another method to calculate the MD5 checksum unless its not possible to do with existing method.

Reply:
My initial version did that, but then I disliked that solution because using a `DigestInputStream` adds so much boilerplate. Furthermore, with the existing solution does not follow proper encapsulation best practice. It's the client that needs to create the `DigestInputStream` rather than `FileContentChecksum`